documentation/Glossary.md

```diff
@@ -28,7 +28,7 @@ In Old IRIMS terms: PROCEDURE.
 
 **Outcome** — The final result of an application. Will most likely be either “Approved” or “Rejected” once it reaches the “Complete” status, and before that would be “Pending” (or perhaps Null).
 
-**Trigger** — A type of Event listener that responds to changes in the database or environment to launch “Actions”. Examples might be “OnSubmit”, “OnReviewSubmit”, “OnApproval”, etc. Can be in response to user-initiated events (like those examples) or time-based, such as schedulers (e.g. if an application must be responded to within a certain time limit, an Action would be Triggered when that time is reached)
+**Trigger** — A type of Event listener that responds to changes in the database or environment to launch “Actions”. Examples might be “OnSubmit”, “ON_REVIEW_SUBMIT”, “OnApproval”, etc. Can be in response to user-initiated events (like those examples) or time-based, such as schedulers (e.g. if an application must be responded to within a certain time limit, an Action would be TRIGGERED when that time is reached)
```

Comment:
Don't need this TRIGGERED, it's just regular prose here.

Global find and replace, huh? ;)

Reply:
Also, “OnSubmit”, “ON_REVIEW_SUBMIT”, “OnApproval” -- 2 didn't get changed -- maybe change them to actual examples, not these obsolete ones.

Reply:
oops missed that capitalise trigger, yeah there were quite a lot that i global replaced and then change after reviewing my own PR, also change to actual trigger examples in this doc
9abb617d4bd0f5005adbbff7f4d7f3db0471ecbe